CreeDictionary/CreeDictionary/templates/CreeDictionary/index.html

```diff
@@ -3,32 +3,41 @@
 {% load static %}
 
 {% block content %}
-    {# tell javascript to display initial lemma #}
-    <data id="initial-lemma-id" value="{{ lemma_id|default:"" }}"></data>
```

Comment:
we no longer need to tell javascript to load the paradigm page

Reply:
GOOD